server-toss/.env.example

```
@@ -15,10 +15,13 @@ TOSS_CLIENT_CERT_PATH=./certs/davinci_new_public.crt
-# 콘솔 인앱광고에 등록한 광고 그룹 ID 콤마 구분
 AD_GROUP_ID_WHITELIST=
-# 콘솔 공유 리워드에 등록한 모듈 ID 콤마 구분 (moduleId가 곧 리워드 ID)
 SHARE_MODULE_ID_WHITELIST=
-EXTERNAL_API=toss|mock
\ No newline at end of file
+TOSS_TEMPLATE_DAILY_PROMPT=
+DAILY_PROMPT_NOTIFICATION_ENABLED=false
+TOSS_TEMPLATE_OVERTAKEN=
+OVERTAKEN_NOTIFICATION_ENABLED=false
+NOTIFICATION_STALE_THRESHOLD_MINUTES=60
+EXTERNAL_API=toss|mock
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXTERNAL_API key should go before the JWT_SECRET key

(UnorderedKey)